geometry_msgs/Pose des_frame
bool run
string goal_tf
string action
string arm